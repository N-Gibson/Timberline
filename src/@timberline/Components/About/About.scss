.about {
  align-items: center;
  display: grid;
  grid-template-columns: 2fr 1fr;
  grid-template-rows: 1fr 1fr;
  grid-template-areas:
    'about about'
    'narrative bucket';
  height: auto;
  text-align: 'left';
}

#about_article {
  grid-area: about;
  display: grid;
  grid-template-columns: 2fr 1fr;
  grid-template-rows: auto;
  justify-items: center;
  align-items: center;
  grid-gap: 2em;
  padding: 2em;
  background-color: #e0e0e0;
  box-shadow: 10px 0px 20px 0px #00000040;
  margin: 0 0 5em 0;
}

#about_banner {
  background-image: url('../../assets/Purple_Aster.jpg');
  background-repeat: no-repeat;
  background-attachment: fixed;
  height: 25rem;
  background-position: center;
  background-size: cover;
  display: flex;
  justify-content: center;
  flex-direction: column;
  margin-top: 120px;
}

#about_banner h2 {
  text-transform: capitalize;
  color: white;
  font-size: 2em;
}

.about_article {
  grid-area: narrative;
  grid-template-columns: 2fr 1fr;
  margin: 2em;
}

.about_article_text {
  background-color: #c8ad96;
  border-radius: 6px;
  box-shadow: 10px 0px 20px 0px #00000040;
  font-size: 1.25em;
  line-height: 2;
  padding: 2em;
}

.about_article_bucket_list {
  background-color: #c8ad96;
  border-radius: 6px;
  box-shadow: 10px 0px 20px 0px #00000040;
  font-size: 1.25em;
  grid-area: bucket;
  line-height: 2;
  padding: 2em;
}

.bucket_list_title {
  border-bottom: 1px solid black;
  font-size: 1.2em;
  margin-bottom: 10px;
}

.bucket_list_ul {
  display: flex;
  flex-direction: column;
  align-items: flex-start;
}

.history {
  height: 75vh;
  display: flex;
  align-items: center;
}

.about_main_image {
  width: 100%;
  border-radius: 6px;
  box-shadow: 10px 0px 20px 0px #00000040;
}
