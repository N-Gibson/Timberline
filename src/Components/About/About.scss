.about {
  align-items: center;
  display: grid;
  grid-template-columns: 2fr 1fr;
  grid-template-areas:
    'narrative bucket';
  height: 75vh;
}

.about_article {
  grid-area: narrative;
  grid-template-columns: 2fr 1fr;
  margin: 2em;
}

.about_article_bucket_list {
  background-color: #C8AD96;
  border-radius: 6px;
  box-shadow: 10px 0px 20px 0px #00000040;
  font-size: 1.25em;
  grid-area: bucket;
  line-height: 2;
  padding: 2em;
}

.about_article_text {
  background-color: #C8AD96;
  border-radius: 6px;
  box-shadow: 10px 0px 20px 0px #00000040;
  font-size: 1.25em;
  line-height: 2;
  padding: 2em;
}

.bucket_list_title {
  border-bottom: 1px solid black;
  font-size: 1.2em;
  margin-bottom: 10px;
}
