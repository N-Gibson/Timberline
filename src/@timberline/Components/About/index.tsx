import React from 'react'

import Hallway from '@timberline/assets/Venado_Hallway.jpg'

import './About.scss'

interface AboutProps {}

const About: React.FC<AboutProps> = () => {
  return (
    <section className="about_main">
      <div className="about_banner_container">
        <div id="about_banner">
          <h2>About Us</h2>
        </div>
      </div>
      <section className="about">
        <article className="about_article" id="about_article">
          <p className="about_article_text">
            Timberline Inc. was founded in 1995 with the goals of creating
            quality solutions in the housing market. It has evolved into a
            company that can do a bit of everything. We take pride in creative 
            problem solving for dynamic construction and repair needs.
          </p>
          <img
            className="about_main_image"
            src={Hallway}
            alt="hallway in house"
          />
        </article>
        <article className="about_article">
          <p className="about_article_text">
            Timberline Inc. is not just about creating quality solutions to
            problems that arise throughout the duration of the home owning
            timeline, it is also about creating an extended family within the
            community. Compassion, empathy, understanding are just some of the
            values that sit at the very core of Timberline Inc. Steve Gibson is
            a creative problem solver that will work with you to find the
            perfect solution. If you would like to work with him, please view
            some of the items on his project bucket list!
          </p>
        </article>
        <article className="about_article about_article_bucket_list">
          <h2 className="bucket_list_title">Bucket List</h2>
          <ul className="bucket_list_ul">
            <li>Work on a home from the 1800's</li>
            <li>Build a cabin in Alaska</li>
            <li>Design and build a liveable tree house</li>
          </ul>
        </article>
      </section>
    </section>
  )
}

export { About }
