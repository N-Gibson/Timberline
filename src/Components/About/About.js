import React from 'react';
import './About.scss';

export const About = () => {
  return (
    <section className="about">
      <article className="about_article">
        <p className="about_article_text">
          Timberline Inc. is not just about creating quality solutions to problems that arise throughout the duration of the home owning timeline, it is also about creating an extended family within the community. Compassion, empathy, understanding are just some of the values that sit at the very core of Timberline Inc. Steve Gibson is a creative problem solver that will work with you to find the perfect solution. If you would like to work with him, please view some of the items on his project bucket list!
        </p>
      </article>
      <article className="about_article">
        <h2 className="bucket_list_title">Bucket List</h2>
        <ul className="bucket_list_ul">
          <li className="bucket_list_li">Work on a home from the 1800's âˆš</li>
          <li className="bucket_list_li">Build a cabin in Alaska</li>
          <li className="bucket_list_li">Design and build a liveable tree house</li>
        </ul>
      </article>
    </section>
  )
}