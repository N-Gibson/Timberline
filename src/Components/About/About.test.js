import React from "react";
import { shallow } from "enzyme";
import About from "./About";

describe("About", () => {
  const wrapper = shallow(<About />);

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
