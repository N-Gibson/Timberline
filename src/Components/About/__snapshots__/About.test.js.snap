// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component Should match snapshot 1`] = `
<Layout
  active="about"
>
  <section
    className="about_main"
  >
    <div
      id="about_banner"
    >
      <h2>
        About Us
      </h2>
    </div>
    <section
      className="about"
    >
      <article
        className="about_article"
        id="about_article"
      >
        <p
          className="about_article_text"
        >
          Timberline Inc. was founded in 1995 with the goals of creating quality solutions in the housing market. It has evolved into a company that can do a bit of everything. Etc... this narrative will need some work.
        </p>
        <img
          alt="hallway in house"
          className="about_main_image"
          src="Venado_Hallway.jpg"
        />
      </article>
      <article
        className="about_article"
      >
        <p
          className="about_article_text"
        >
          Timberline Inc. is not just about creating quality solutions to problems that arise throughout the duration of the home owning timeline, it is also about creating an extended family within the community. Compassion, empathy, understanding are just some of the values that sit at the very core of Timberline Inc. Steve Gibson is a creative problem solver that will work with you to find the perfect solution. If you would like to work with him, please view some of the items on his project bucket list!
        </p>
      </article>
      <article
        className="about_article about_article_bucket_list"
      >
        <h2
          className="bucket_list_title"
        >
          Bucket List
        </h2>
        <ul
          className="bucket_list_ul"
        >
          <li>
            Work on a home from the 1800's âˆš
          </li>
          <li>
            Build a cabin in Alaska
          </li>
          <li>
            Design and build a liveable tree house 
          </li>
        </ul>
      </article>
    </section>
  </section>
</Layout>
`;
